x/auth/migrations/v6/migrate.go

```diff
@@ -0,0 +1,12 @@
+package v6
+
+import (
+	"context"
+)
+
+type migrateAccNumFunc = func(ctx context.Context) error
+
+// Migrate account number from x/auth account number to x/accounts account number
+func Migrate(ctx context.Context, f migrateAccNumFunc) error {
+	return f(ctx)
```

Comment:
personal preference: this adds just the v6 namespace while the impl is in the migrator. Why not move the `v56SetAccounts` code into this method so that you have it separated. You can have better interfaces and tests as  the scope is very limited. It would also be clear to others reading the code, what happens in v6 rather than searching for the method passed.

Reply:
yeah i dont know why i'm not just move it in the Migrate func directly tbh, probably the case of late night coding